zoo/policies/human_in_the_loop.py

```diff
@@ -17,13 +17,6 @@
 from smarts.core.agent_interface import AgentInterface, AgentType
 from smarts.core.utils.episodes import episodes
```

Comment:
This import can also be removed.
```suggestion
```

Please also remove the `gym` and `logging` imports.